### ISSUE TITLE
[BUG] [Linter] T038 false positive on inline comment

### ISSUE THREAD

**Issue by ziima** [djlint/djLint#2297]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: debian unstable
- Python Version (`python --version`): 3.13.14
- djLint Version (`djlint --version`): 1.42.3
- template language: jinja

## Issue

A false positive T038 in case the `if` block is followed by an inline comment using `{# ... #}`.

## How To Reproduce

MWE:
```
{% if my_variable %}{# Comment is here #}
text
{% endif %}
```

When linted as
```djlint --profile=jinja --lint t038-false-positive.txt```

Returns
```
Linting 1/1 files [####################################]


t038-false-positive.txt
──────────
T038 3:0 End tag has no matching block tag. {% endif %}

Linted 1 file, found 1 error.
```

The template passes linting when the comment is removed.

## Contents of djlint.toml/.djlintrc/pyproject.toml [tool.djlint]

No file
